src/routes.ts

```diff
@@ -1,42 +1,40 @@
+import { getPreferenceValues, Cache } from "@raycast/api";
+import { promises as fs } from "fs";
+import os from "os";
+import path from "path";
+
 export interface Route {
   name: string;
   url: string;
   routes: Array<Route>;
 }
 
-export const routes: Array<Route> = [
-  {
-    name: "gh (GitHub)",
-    url: "https://github.com",
-    routes: [
-      {
-        name: "docs",
-        url: "https://github.com/camunda/camunda-docs",
-        routes: [
-          {
-            // gh docs issues
-            // gh docs pull/123
-            name: "*",
-            url: "https://github.com/camunda/camunda-docs/${0}",
-            routes: [],
-          },
-          {
-            name: "pulls",
-            url: "https://github.com/camunda/camunda-docs/pulls",
-            routes: [],
-          },
-        ],
-      },
-      {
-        name: "me",
-        url: "https://github.com/pepopowitz",
-        routes: [],
-      },
-    ],
-  },
-  {
-    name: "google",
-    url: "https://google.com",
-    routes: [],
-  },
-];
+interface Preferences {
+  goto_path: string;
+}
+
+export async function getRoutes(): Promise<Array<Route>> {
+  const cache = new Cache();
+  const cached = cache.get("routes");
+  if (cached) {
+    return Promise.resolve(JSON.parse(cached));
+  }
+
+  const routes = await loadRoutesFromFile();
+  cache.set("routes", JSON.stringify(routes));
+  return Promise.resolve(routes);
+}
```

Comment:
Still not sure if this is the way raycast would want me to do it, but to avoid hitting the file system every time the command runs, I took advantage of the [Cache](https://developers.raycast.com/api-reference/cache) feature to store the routes in memory. And then the new `reload` command will bust the cache, if I change the config file and need to reload it.